OPHD/States/CrimeRateUpdate.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "../Map/Tile.h"
+#include <vector>
+
+class PopulationPanel;
+
+namespace CrimeRate
+{
+	// Returns change in current morale caused by colony's mean crime rate
+	int update(const std::vector<TileList>& policeOverlays, PopulationPanel& populationPanel);
+}
```

Comment:
The meaning of the `int` return type isn't super obvious for a function named `update`. I suspect if this was named in a more descriptive manner, the namespace could be dropped as well.

Reply:
Yeah, I agree the return isn't very obvious without the comment. I am having trouble deciding how this module should pass the change in morale back to the main game state to make it less confusing.

Reply:
This could probably be done fairly cleanly by declaring a module level variable, say, CrimeChange or MoraleChange or whatever you want to call it, put it in an unnamed namespace and then provide a get/set type of function. Since the CrimeRateUpdate isn't a distinct class this is probably the cleanest way to do it.

Actually it makes me want to change the way Population handles food consumption -- it current returns 'actual food consumed' in the update() function which should probably be a separate function. Classic case of a single function trying to do more than one thing.